packages/ui-components/src/components/Matrix/Matrix.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useEffect, useReducer, useRef } from 'react';
+import styled from 'styled-components';
+import { Table, TableBody } from '@material-ui/core';
+import { MatrixHeaderRow } from './MatrixHeaderRow';
+import { MatrixColumnType, MatrixRowType } from '../../types';
+import { hexToRgba } from './utils';
+import { ACTION_TYPES, MatrixContext, MatrixDispatchContext, matrixReducer } from './MatrixContext';
+import { MatrixNavButtons } from './MatrixNavButtons';
+import { MatrixRow } from './MatrixRow';
+import { EnlargedMatrixCell } from './EnlargedMatrixCell';
+
+const MatrixTable = styled.table`
+  border-collapse: collapse;
+  border: 1px solid ${({ theme }) => hexToRgba(theme.palette.text.primary, 0.2)};
```

Comment:
I'm keen to remove the `hexToRgba` function in favour of just using hex colors with opacity. So for example we should just be able to do `${theme.palette.text.primary}33` to get the same result, right?.
https://www.w3schools.com/cssref/css_colors_legal.php